src/fundus/publishers/de/__init__.py

```diff
@@ -202,6 +203,18 @@ class DE(PublisherEnum):
         parser=WAZParser,
     )
 
+    Heise = PublisherSpec(
+        name="Heise",
+        domain="https://www.heise.de",
+        sources=[
+            RSSFeed("https://www.heise.de/rss/heise.rdf"),
+            Sitemap("https://www.heise.de/sitemapindex.xml"),
+            NewsMap("https://www.heise.de/news/sitemap.xml"),
```

Comment:
Filtering out the bloat from the sitemap actually results in a really, really bad runtime. I cancelled it every time, because it took more than a minute to load a single article. What would you suggest is the better option: just removing the sitemap and relying on the content of the RSS feed or maybe, instead of the index, using just this: https://www.heise.de/sitemap.xml it actually doen't look like bloat to me, but like an unsorted list of the recent articles